webextensions/manifest/permissions.json

```diff
@@ -220,9 +220,9 @@
             }
           }
         },
-        "contextMenu": {
+        "contextMenus": {
           "__compat": {
-            "description": "<code>contextMenu</code>",
+            "description": "<code>contextMenus</code>",
```

Comment:
This line is unnecessary as the renderer defaults to <code>\`&lt;code&gt;${name}&lt;/code&gt;\`</code>.

Reply:
Would it be okay for you if I made a separate PR for this? This way I could remove all of these lines at once.

Reply:
> This line is unnecessary as the renderer defaults to `<code>${name}</code>`.

This is down to me.

Some of these subfeatures need `description`, because the string uses characters that aren't allowed in identifiers (e.g. ["downloads.open"](https://github.com/mdn/browser-compat-data/blob/master/webextensions/manifest/permissions.json#L358)). I asked for `description` to be added to all of them, to be consistent (this is after all a string value, not an identifier).

I don't feel very strongly about it, so it's considered a bad idea and worth the time of fixing, we can fix it for all keys (except obviously things like "downloads.open"). But I agree that this should be a separate PR.